app/models/container_node.rb

```diff
@@ -75,6 +75,12 @@ def ipaddress
   end
 
   def cockpit_url
-    URI::HTTP.build(:host => ipaddress, :port => 9090)
+    if MiqRegion.my_region.role_active?("cockpit_ws")
+      uri = URI("/dashboard/cockpit_redirect")
```

Comment:
We probably want a module with these cockpit integration constants in case we need to use them elsewhere or need to change them in the future.  Maybe the MiqCockpit module?